### ISSUE TITLE
Some Channels Videos Are Missing From Deck

### ISSUE THREAD

**Issue by Kyzoku** [NabokD/pockettube#48]
There are videos missing from the deck for specific channels.

One specific channel that I noticed missing videos from is "https://www.youtube.com/@CeresFauna".  Most of this channels videos are located in the Live tab, but the deck doesn't pick up on these.  I've checked this on a number of channels that operate similarly and their videos are missing from the deck as well.  I've made sure that no videos are being filtered or hidden.

I just recently purchased a year subscription and now I'm a little disappointed that there are videos missing from the deck.  The deck was the primary reason for my subscription and this just reduced the usefulness of the extension for me by quite a bit.  I really hope I'm missing a setting somehow or that this is an oversight that gets rectified soon.

Edit:
I checked this on both Microsoft Edge and Vivaldi browsers.

**Comment by NabokD** [comment]
Hello, yes, Deck is supported only Videos tab. The live tab is new for YouTube and is not supported by Deck yet. The feature is on the list of investigating, however, it seems not easy to collect videos from different places is not easy to do.

As a workaround, you can create a separate group with the live channels and add the group to Deck. Group uses a different way to create a feed and includes live videos too.

**Comment by Kyzoku** [comment]
It seems that will have to do for now, but the group seems to not have a load more button.

**Comment by NabokD** [comment]
Yes, groups feed consist of 15 latest channel from every channelThe group's feed consists of 15 latest videos from every channel. YouTube restricts it, so the channel feed uses all videos from the Videos tab to load all videos. I'm unsure it's possible to mix Videos and Live videos tabs on Deck. It will be followed issue with sorting...

**Comment by chickenSandwich** [comment]
Does this account for channel's that only have playlists?  

This channel: https://www.youtube.com/@sciencefiction1167 when I add it to any subscription group, is not displayed in the group.  (If this is a separate issue, I can create a new issue).

**Comment by NabokD** [comment]
It's expected for the channel, because it's without video, and we consider the channel as having died and not showing in the list
